M2/Macaulay2/packages/MultigradedImplicitization.m2

```diff
@@ -10,7 +10,7 @@ newPackage(
     Email => "benhollering@gmail.com",
     HomePage => "https://sites.google.com/view/benhollering"}
   },
-  Headline => "A package for levaraging multigradings to solve implicitization problems",
+  Headline => "solving implicitization problems using multigradings",
```

Comment:
I'm trying to make all package headlines uniform, which includes removing redundant clauses. @bkholler is this change okay with you?